README.md

```diff
@@ -25,6 +25,16 @@ or
 
 `yarn add graphql-hooks`
 
+## Support
+
+- Node.js v8.5+
+- Browsers [`> 1%, not dead`](https://browserl.ist/?q=%3E+1%25%2C+not+dead)
+
+Consider polyfilling:
+
+- [`Promise`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+- [`fetch`](https://developer.mozilla.org/docs/Web/API/Fetch_API)
```

Comment:
We should link to the `GraphQLClient` docs where the user can pass in the `fetch` implementation instead of polyfilling.